Update to permissions requested by APP (#4139)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/information-barriers-policies.md

   2. When prompted, sign in using your work or school account for Office 365.

>    3. In the **Permissions requested** dialog box, review the information, and then choose **Accept**. The permissions reqested by the App is given below <add a screenshot>

Reviewer: ```suggestion
   3. In the **Permissions requested** dialog box, review the information, and then choose **Accept**. The permissions requested by the App are given below. 
  add a screenshot>
```

>    3. In the **Permissions requested** dialog box, review the information, and then choose **Accept**. The permissions reqested by the App is given below <add a screenshot>

Reviewer: ```suggestion
   3. In the **Permissions requested** dialog box, review the information, and then choose **Accept**. The permissions requested by the app are given below.
   <add a screenshot>
```
